toxav/groupav.c

```diff
@@ -56,13 +56,15 @@ static Group_JitterBuffer *create_queue(unsigned int capacity)
         size *= 2;
     }
 
-    Group_JitterBuffer *q;
+    Group_JitterBuffer *q = (Group_JitterBuffer *)calloc(sizeof(Group_JitterBuffer), 1);
 
-    if (!(q = (Group_JitterBuffer *)calloc(sizeof(Group_JitterBuffer), 1))) {
+    if (!q) {
         return nullptr;
     }
 
-    if (!(q->queue = (Group_Audio_Packet **)calloc(sizeof(Group_Audio_Packet *), size))) {
```

Comment:
What are you some kinda moron who doesn't like assigning variables within if statements

Reply:
If this question is serious, let me know and I'll be happy to talk about it.